src/libs/actions/Report.js

```diff
@@ -328,15 +331,24 @@ function fetchChatReportsByIDs(chatList) {
  *
  * @param {Number} reportID
  * @param {Number} sequenceNumber
+ * @param {Boolean} saveNewMarkerPosition
  */
-function setLocalLastRead(reportID, sequenceNumber) {
+function setLocalLastRead(reportID, sequenceNumber, saveNewMarkerPosition) {
```

Comment:
I would separate this logic from here since it adds unnecessary complexity. 

If we want to perform a `setNewMarkerPosition()` then it is just as easy to call  a method named `setNewMarkerPosition()` after calling `setLocalLastRead()`.

Reply:
Sorta? There is also a difference to what we set `unreadActionCount` to, though, so I think the if/else makes sense

Reply:
>There is also a difference to what we set unreadActionCount 

Yeah, this part is also confusing tbh. It's unclear why calling this method would set that to `0` as the default case?

Reply:
Resolving this to keep the discussion in a single thread